state/presence/presence.go

```diff
@@ -383,15 +381,20 @@ type pingInfo struct {
 }
 
 func (w *Watcher) findAllBeings() (map[int64]beingInfo, error) {
-	beings := make([]beingInfo, 0)
 	session := w.beings.Database.Session.Copy()
 	defer session.Close()
-	beingsC := w.beings.With(session)
 
-	err := beingsC.Find(bson.D{{"model-uuid", w.modelUUID}}).All(&beings)
+	beingsC := w.beings.With(session)
+	t0 := time.Now()
+	beings := make([]beingInfo, 0)
+	err := beingsC.Find(bson.M{
+		"_id": bson.M{"$regex": bson.RegEx{"^" + w.modelUUID, ""}},
+	}).All(&beings)
```

Comment:
given this table is larger than we think it is, should we be actually using BatchSize and Iter() to load it into the map?